rust/package/agama.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Fri May  2 07:42:21 UTC 2025 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Update dependencies (gh#agama-project/agama#2317).
```

Comment:
I wonder why to add this in the changelog. I'm not against, but not sure how helpful it is from a change log perspective if it does not introduce a breaking change or so.

Reply:
Just in case something goes wrong later, I would expect this to make it easier to track (especially for somebody who has not been involved).